src/coreclr/inc/clrconfigvalues.h

```diff
@@ -677,6 +677,11 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeCircularMB, W("EventPipeCircularMB"),
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeProcNumbers, W("EventPipeProcNumbers"), 0, "Enable/disable capturing processor numbers in EventPipe event headers")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeOutputStreaming, W("EventPipeOutputStreaming"), 0, "Enable/disable streaming for trace file set in COMPlus_EventPipeOutputPath.  Non-zero values enable streaming.")
 
+#ifdef FEATURE_AUTO_TRACE
+RETAIL_CONFIG_DWORD_INFO(INTERNAL_AutoTrace_N_Tracers, W("AutoTrace_N_Tracers"), 0, "")
```

Comment:
@josalem Should this be a retail enabled thing, or this this just slip?

Reply:
I believe it must be retail given it was always requested previously. Or perhaps the entire AutoTrace feature is only enabled on Debug?

Reply:
It's not enabled - you need to add a flag manually to compile.

Reply:
Ah, I see - The getenv used to not get compiled before. So this complus setting never made it to a public build and this just added it to CLRConfigs, but would have no effect. .

Reply:
But if it is ever compiled in Release mode this needs to be a `RETAIL_` configuration. Unless there is some guarantee that this feature is only for CHK and DBG I am inclined to leave it.

Reply:
This feature is _only_ for CHK/DBG. I'm not certain we actually need it anymore with the introduction of the diagnostic ports mechanism. I might spin a PR to remove the code.